internal/k8s/ako_init.go

```diff
@@ -273,6 +275,10 @@ func (c *AviController) InitController(informers K8sinformers, registeredInforme
 	fastRetryQueue := utils.SharedWorkQueue().GetQueueByName(lib.FAST_RETRY_LAYER)
 	fastRetryQueue.SyncFunc = SyncFromFastRetryLayer
 	fastRetryQueue.Run(stopCh, fastretrywg)
+
+	slowRetryQueue := utils.SharedWorkQueue().GetQueueByName(lib.SLOW_RETRY_LAYER)
+	slowRetryQueue.SyncFunc = SyncFromSlowRetryLayer
+	slowRetryQueue.Run(stopCh, slowretrywg)
```

Comment:
we'll need a slowRetryQueue.StopWorkers(stopCh) on L297 as well?

Reply:
done. great catch.

Reply:
thanks